virtwho/virt/ahv/ahv.py

```diff
@@ -149,15 +151,15 @@ def getHostGuestMapping(self):
             else:
                 self.logger.debug("Host '%s' doesn't have any vms", host_uuid)
 
-            cluster_uuid = self._interface.get_host_cluster_uuid(host)
+            cluster_name = self._interface.get_host_cluster_name(host, cluster_ids)
```

Comment:
Is it really possible that the cluster_name is `None`? What should we do in this case? If it is possible then we probably should not add the `hypervisor.cluster` to the dictionary of facts. Thoughts?

Reply:
Perhaps something like UUID:[uuid]. It has a cluster with an id, but no textual name.

Reply:
When the cluster has only UUID but it does not have name, then it is OK to print some debug message about it and use UUID for `hypervisor.cluster`.

When the `cluster_name` is None, then `hypervisor.cluster` should not be added to dictionary.